.github/workflows/continuous-integration-workflow.yml

```diff
@@ -24,18 +24,20 @@ jobs:
           distribution: ${{ matrix.distribution }}
           cache: maven
       - name: Build with Maven
-        run: |
-          if [[ ${{ github.ref_name }} == 'main' ]]; then            
-            ./mvnw -B package -Pcoverage --file pom.xml
-          else
-            ./mvnw -B package
-          fi
-      - name: Upload failed test reports
-        uses: actions/upload-artifact@v3
-        if: always()
-        with:
-          name: surefire-reports_java${{ matrix.java }}
-          path: xmppserver/target/surefire-reports
+        run: ./mvnw -B package -Dmaven.test.skip=true --file pom.xml
+        # RESTORE THIS! The above is adjusted only to speed up development.
```

Comment:
Does this need to come back then?

Reply:
This is still something that we're playing with. I've marked the PR as being a draft now, to prevent confusion.